storybookjs/storybook
# Core: Enable gzip compression on the development server

Issue: #14280 

## What I did

This PR enables compression on assets served by the dev server. The compression level is low, which yields good compression results without slowing down the feedback loop between saving a file and changes appearing in the browser.

This is useful when a Storybook has large bundles.

## How to test

Make sure HMR works as expected. It should be okay, but in older versions of the compression package, it could sometimes be an issue.

Also ensure loading and reloading performance is within acceptable parameters.
